docs/rule/data-attributes-required-default.md

```diff
@@ -0,0 +1,37 @@
+# data-attributes-required-default
+
+:white_check_mark: The `extends: 'recommended'` property in a configuration file enables this rule.
+
+Require the presence of a default value for data attributes.
```

Comment:
What does "default" mean here? Do you just mean that they need a value?

Reply:
correct, so instead of positional params they would be named arguments.

Reply:
> correct, so instead of positional params they would be named arguments.

They are **never** positional params (or named arguments) in these examples. They are attributes.